src/pages/patientView/copyNumberAlterations/column/AnnotationColumnFormatter.tsx

```diff
@@ -50,23 +51,18 @@ export default class AnnotationColumnFormatter
     }
 
    /**
-     * Returns an ICivicEntry if the civicGenes and civicVariants have information about the gene and the mutation (variant) specified. Otherwise it returns 
-     * an empty object.
-     */
+    * Returns an ICivicEntry if the civicGenes and civicVariants have information about the gene and the mutation (variant) specified. Otherwise it returns 
+    * an empty object.
+    */
     public static getCivicEntry(copyNumberData:DiscreteCopyNumberData[], civicGenes:ICivicGene, civicVariants:ICivicVariant): ICivicEntry | null
     {
         let geneSymbol: string = copyNumberData[0].gene.hugoGeneSymbol;
         let geneVariants: {[name: string]: ICivicVariantData} = civicVariants[geneSymbol];
         let geneEntry: ICivicGeneData = civicGenes[geneSymbol];
         let civicEntry = null;
         //Only return data for genes with variants or it has a description provided by the Civic API
-        if (geneVariants ||
-               (geneEntry && geneEntry.description != "")) {
-            civicEntry = {name: geneEntry.name,
-                             description: geneEntry.description,
-                             url: geneEntry.url,
-                             variants: geneVariants,
-                            }
+        if (geneVariants || (geneEntry && geneEntry.description != "")) {
```

Comment:
Why do we need an extra `geneEntry` check here, but not within the other `getCivicEntry` function (the one in `mutationTable/column/AnnotationColumnFormatter`)?

If we can use the same condition for both functions (i.e: if we can add the `geneEnrty` check for the other function as well), then we can move even more functionality into the `buildCivicEntry` function. Well, basically everything except the first line where we get the gene symbol.